client/elife-theme/src/icons/material/Profile.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+const Profile = props => (
+  <svg height="24px" viewBox="0 0 24 24" width="24px" {...props}>
+    <path d="M12 5.9a2.1 2.1 0 1 1 0 4.2 2.1 2.1 0 0 1 0-4.2m0 9c2.97 0 6.1 1.46 6.1 2.1v1.1H5.9V17c0-.64 3.13-2.1 6.1-2.1M12 4C9.79 4 8 5.79 8 8s1.79 4 4 4 4-1.79 4-4-1.79-4-4-4zm0 9c-2.67 0-8 1.34-8 4v3h16v-3c0-2.66-5.33-4-8-4z" />
+    <path d="M0 0h24v24H0z" fill="none" />
```

Comment:
I'm guessing we want fill to be white?

Designs:
![image](https://user-images.githubusercontent.com/15656538/46945084-cfbabd00-d06b-11e8-8510-88f40a584b02.png)


Actual:
![image](https://user-images.githubusercontent.com/15656538/46945051-c0d40a80-d06b-11e8-94a1-3c345f32493c.png)

Reply:
Yeah, my (edited) [question on the ticket](https://github.com/elifesciences/elife-xpub/issues/777#issuecomment-429310826) got missed. Have created new issue #795.